maxim/logger/agno/client.py

```
@@ -0,0 +1,443 @@
+        created_at = agno_response.get("created_at", int(time.time()))
+        context_rules = agno_response.get("context_rules", {})
+
+        # Token usage
+        def _first(val):
+            if isinstance(val, list) and val:
+                return val[0]
+            if isinstance(val, int):
+                return val
+            return 0
+
+        prompt_tokens = _first(metrics.get("prompt_tokens", 0))
```

Ruff (0.11.9)

Local variable `prompt_tokens` is assigned to but never used

Remove assignment to unused variable `prompt_tokens`

(F841)